locale/es_ES/locale.xml

```diff
@@ -14,8 +14,20 @@
 
 <locale name="es_ES" full_name="Español (España)">
 	<message key="plugins.generic.paperbuzz.displayName">Módulo Paperbuzz</message>
-	<message key="plugins.generic.paperbuzz.description">Módulo para mostrar métricas a Nivel de Artículo (Paperbuzz, Article Level Metrics). Para usar el servicio Paperbuzz, necesitas disponer de DOIs.</message>
+	<message key="plugins.generic.paperbuzz.description">Módulo para mostrar la Métrica a Nivel de Artículo (Paperbuzz, Article Level Metrics). Para usar el servicio Paperbuzz, necesitas disponer de DOIs.</message>
```

Comment:
Replace "la Métrica" for "la métrica". Capital letter is not need.
In spanish, "Nivel de Artículo" also need to be without capitals.